src/mixxx.cpp

```diff
@@ -857,8 +867,15 @@ void MixxxMainWindow::onNewSkinLoaded() {
                              ConfigKey("[EffectRack1]", "show"));
     setVisibilityOptionState(m_pViewShowCoverArt,
                              ConfigKey("[Library]", "show_coverart"));
+
+    ControlObjectSlave* pCOShowSamplers = new ControlObjectSlave(
```

Comment:
You will create a new ControlObjectSlave every time you change the skin.
I think the   ControlObjectSlave holds a strong reference to the ControlObject, so it is not deleted when the skin has no  "[Library]", "show_coverart". (Please verify it) 
Since the grey-out logic checks for the existence of the CO, it will fail now. 
You probably need to delete the ControlObjectSlave before starting to instantiate a new skin.